### PR TITLE
[MediaBundle] Keep sidebar on upload and bulk upload

### PR DESCRIPTION
**Pull request by cv65kr**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Improvment
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | No

### CODE REVIEW

File: src/Kunstmaan/MediaBundle/Resources/views/Media/bulkUpload.html.twig [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+{% block body %}
+    {% set lowestTopChild = foldermanager.getRootFolderFor(folder) %}
+    {% if lowestTopChild %}
+        {% set navigationChildren = foldermanager.getFolderHierarchy(lowestTopChild) %}
+    {% endif %}
+    {{ parent() }}
+{% endblock %}
+
+
+{% block sidebar %}
```
Comment (sandergo90) [inline src/Kunstmaan/MediaBundle/Resources/views/Media/bulkUpload.html.twig]: Maybe we should add a new layout twig file for this. I can see that there are already 3 twig files with the sames blocks in it. Maybe put them in a twig file and extend from that template.

**Comment by cv65kr** [comment]
@sandergo90 done

**Comment by sandergo90** [comment]
@cv65kr Can you also change the other templates? Because there is more then 1 template that contains the sidebar. They need to extend the new sidebar template also.

**Comment by Devolicious** [comment]
Hi @cv65kr, due to inactivity and the fact this PR still needs some work I'm gonna close it. You can always reopen and finish the work needed.

**Pull request closed without merging**